css/style.css

```
@@ -0,0 +1,235 @@
+@charset "utf-8";
+
+
+/****************** heather *********************/
+header{
+  position:fixed;
+  top:0;
+  left:0;
+  right:0;
+  padding:10px 200px;
+  height:70px;
+  display:flex;
+  justify-content:space-between;
+  align-items:center;
+  background-color:#fff;
+  border-bottom:1px solid #dfdfdf;
+}
+header button{
+  padding:120x 20px;
+  display:block;
+  width:128px;
+  height:48px;
+  border-radius:8px;
+}
+/****************** CSS seting *********************/
+.container{
+  margin:0 auto;
+  width:100%;
+  max-width:1200px;
+}
+button{
+  cursor:pointer;
+}
+:root{
+    --brand_blue:#3692FF;
+    --brand_sky:#CFE5FF;
+    --brand_sky_light:#E6F2FF;
+    --hover_dark_blue:#1967D6;
+    --cool_gray:#374151;
+    --cool_tray_lignt:#F3F4F6;
+}
+.brand_b{
+  background-color:var(--brand_blue);
+  color:#fff;
+  transition:all 0.5s;
+}
+.brand_b:hover{
+  background-color:var(--hover_dark_blue);
+}
+
+/****************** main *********************/
+.main_banner{
+  width:100%;
+  height:540px;
+  background-color:var(--brand_sky);
+  background-image:url(.././images/Img_home_top.png);
```

중간에 `./`은 없어도 될 것 같습니다~
```suggestion
  background-image:url(../images/Img_home_top.png);
```